logsearch-for-cloudfoundry-boshrelease/templates/stub.yml

```diff
@@ -20,18 +20,15 @@ properties:
     oauth2_client_id: "kibana_admin"     
     oauth2_client_secret: "keepitsimple" 
   redis:
-    host: 10.244.10.10
+    host: "10.244.11.10"
   cloudfoundry:
-    cloudControllerUri: "https://api.10.244.0.34.xip.io"
-    uaaUri: "https://uaa.10.244.0.34.xip.io"
+    system_domain: "workspace.pcf.stayup.io"
```

Comment:
Oops.  The field names are correct.  But I shouldn't have committed my internal settings